core/src/test/scala/com/salesforce/op/filters/FeatureDistributionTest.scala

```diff
@@ -107,15 +119,15 @@ class FeatureDistributionTest extends FlatSpec with PassengerSparkFixtureTest wi
       else d.distribution.length shouldBe 2
     }
     distribs(0).nulls shouldBe 0
-    distribs(0).summaryInfo should contain theSameElementsAs Array(0.0, 1.0)
+    distribs(0).summaryInfo should contain theSameElementsAs Array(0.0, 2.0, 100.0, 10.0)
```

Comment:
Same here, why did the max change?

Reply:
this one reads the info from hardcoded 'summaries' above